### PR TITLE
docs(cookbook): Add simple OSM example.

### PR DESCRIPTION
**Pull request by bradh**

### CODE REVIEW

File: docs/cookbook/osm/index.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  {
+    "providers": {
+      "basemap": {
+        "maps": {
+          "osm": {
+            "title": "OpenStreetMap",
+            "type": "BaseMap",
+            "baseType": "XYZ",
+            "provider": "OSM",
+            "url": "//c.tile.openstreetmap.org/{z}/{x}/{y}.png",
```
Comment (wallw-bits) [inline docs/cookbook/osm/index.rst]: Does OSM have `//[a-c].tile.openstreetmap.org/`? If so, we should use more than just `c`.

I think single URL templating is currently broken in OpenSphere (and we should write an issue for that), but you can do:

```
"urls": [
  "//a.tile.openstreetmap.org/{z}/{x}/{y}.png",
  "//b.tile.openstreetmap.org/{z}/{x}/{y}.png",
  "//c.tile.openstreetmap.org/{z}/{x}/{y}.png"
]
```

Comment (bradh) [reply docs/cookbook/osm/index.rst]: Yeah, it does. I'll test templating (didn't see any code to support it), and raise the ticket as appropriate.

Will update the PR to include multiple URLs.

Comment (bradh) [reply docs/cookbook/osm/index.rst]: Actually, it looks like URL array is also a little tricky (#86). Will see if I can fix that up.

Comment (bradh) [reply docs/cookbook/osm/index.rst]: Urgh. `urls`, not `url`....

**Pull request merged**